lyngk_board.js

```diff
@@ -0,0 +1,210 @@
+var lyngkModule = (function() {
+  my = {};
+  var verticalX = [1, 2, 3, 4, 5, 6, 7];

```

Comment:
These names could use work.  Try to find a more intuitive description without getting wordy!